File: OneCodeTeam/How to use DropDownList with ASP.NET MVC/[C#]-How to use DropDownList with ASP.NET MVC/C#/CSDropdownListMVC4/Models/CascadingDropDownSampleModel.cs
#region "Directives"

using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

#endregion

namespace CSDropdownListMVC4.Models
{
    /// <summary>
    /// This model is used to bind the CascadingDropDownSample Controller's Index action
    /// </summary>
    public class CascadingDropDownSampleModel
    {
        /// <summary>
        /// Dictionary holding the sample Makes values
        /// </summary>
        public IDictionary<string, string> Makes { get; set; }
    }
}

File: OneCodeTeam/How to use DropDownList with ASP.NET MVC/[VB]-How to use DropDownList with ASP.NET MVC/VB/VBDropdownListMVC4/Models/CascadingDropDownSampleModel.vb
''' <summary>
''' This model is used to bind the CascadingDropDownSample Controller's Index action
''' </summary>
Public Class CascadingDropDownSampleModel
    ''' <summary>
    ''' Dictionary holding the sample Makes values
    ''' </summary>
    Public Property Makes() As IDictionary(Of String, String)
        Get
            Return m_Makes
        End Get
        Set(value As IDictionary(Of String, String))
            m_Makes = Value
        End Set
    End Property

    Private m_Makes As IDictionary(Of String, String)
End Class
